OJP/OJP_Trips.xsd

```diff
@@ -117,6 +130,21 @@
 					<xs:documentation>For each mode in this list a separate monomodal trip shall be found - in addition to inter-modal solutions.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="PassengerCategory" type="PassengerCategoryEnumeration" minOccurs="0" maxOccurs="unbounded">
+				<xs:annotation>
+					<xs:documentation>Sequence of all passenger categories, for which this search shall be conducted. In rare cases additional offerings may be available (e.g., demand responsive service with certain areas being reserved for seniors).</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="AccessModeList" type="AccessModesListOfEnumerations" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Access modes that are available to the traveller for the trip. The requester can tell the system what he might use as an access mode (e.g., car, motorcycle, ski) and it's the system's responsibility to know which access modes are supported or required by each service. E.g., if one goes by car, then only ferry and carTransportRail may be possible for public transport. If the traveller has skis or a snowboard with him, then dragLifts can show up in the results to a search. When nothing is provided in the time table, then foot and bicycle should be assumed by the trip planner. In the end the provision of the list does not guarantee valid results in all circumstances. E.g. the traveler may have left the car, went with the skis up the hill in a dragLift and can't access the caron the other side of the mountain for the next leg with carTransportRail. So the enquirer always needs to study the trip results to see what AccessMode and/or are provided there. The implementation might also have an internal mapping which AccessMode can "carry" another one (e.g. a car can carry skis).</xs:documentation>
```

Comment:
```suggestion
					<xs:documentation>Access modes that are available to the traveller for the trip. The requester can tell the system what he might use as an access mode (e.g., car, motorcycle, ski) and it's the system's responsibility to know which access modes are supported or required by each service. E.g., if one goes by car, then only ferry and carTransportRail may be possible for public transport. If the traveller has skis or a snowboard with him, then dragLifts can show up in the results to a search. When nothing is provided in the time table, then foot and bicycle should be assumed by the trip planner. The resulting trips satisfy the following conditions: (a) the PT legs of the trip are possible for all access modes in the list, (b) the non-PT legs are possible by foot or any of the access modes in the list. In the end the provision of the list does not guarantee valid results in all circumstances. E.g. the traveller may have left the car, went with the skis up the hill in a dragLift and can't access the car on the other side of the mountain for the next leg with carTransportRail. So the enquirer always needs to study the trip results to see what AccessMode and/or are provided there. The implementation might also have an internal mapping which AccessMode can "carry" another one (e.g. a car can carry skis).</xs:documentation>
```

Reply:
<xs:documentation>Access modes that are available to the traveller for the trip. The requester can tell the system what he might use as an access mode (e.g., car, motorcycle, ski) and it's the system's responsibility to know which access modes are supported or required by each service. E.g., if one goes by car, only then car ferry and carTransportRail may be possible for public transport. If the traveller has skis or a snowboard with him, then dragLifts can show up in the results to a search. When nothing is provided in the time table, then foot and bicycle should be assumed by the trip planner. The resulting trips satisfy the following conditions: (a) the PT legs of the trip are possible for all access modes in the list, (b) the non-PT legs are possible by foot or any of the access modes in the list. In the end the provision of the list does not guarantee valid results in all circumstances. E.g. the traveller may have left the car, went with the skis up the hill in a dragLift and can't access the car on the other side of the mountain for the next leg with carTransportRail. So the enquirer always needs to study the trip results to see what AccessMode and/or are provided there. The implementation might also have an internal mapping which AccessMode can "carry" another one (e.g. a car can carry skis).</xs:documentation>

Request AccessMode: AccessMode the traveler can offer at the origin 

 [foot car ski] 
 
 examples:
 * (walk - )car - carTransportRail - car - walk - dragLift - walk
											-> losing car      -> [foot ski]
 *  (walk -) dragLift - walk - carTransportRail - walk  (nonsense)
 
Delivery AccessMode: AccessMode that are supported by a SITE.


BLS: bicycle motorcycle car (truck)
MGB:  (foot) (motorcycle) car (truck)
RhB:   motorcycle car
HuPac:  truck

Mapping
ski -> car
foot -> car
bicycle -> car

Norman: The need in the distributed manner. More generalized about context to be provided in the DJP.